src/environment/interval_adjustor.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABCMeta, abstractmethod
+from time import perf_counter
+
+
+class IntervalAdjustor(metaclass=ABCMeta):
+    def __init__(self, interval: float) -> None:
+        self._start_time = perf_counter()
```

Comment:
こっちの書き方の方が良いと言えば、良いです。
```suggestion
import math
class IntervalAdjustor(metaclass=ABCMeta):
    _start_time: float = -math.inf
    def __init__(self, interval: float) -> None:
        self.reset()
```